app/src/main/res/values/colors.xml

```diff
@@ -29,4 +29,5 @@
     <color name="color_yellow_200">#FFFF9D</color>
     <color name="color_black_transprarent">#B2000000</color>
     <color name="color_white_transparent">#ccffffff</color>
+    <color name="color_green_accent_400">#14cc75</color>
```

Comment:
WARNING: [UnusedResources] The resource `R.color.color_green_accent_400` appears to be unused
 Unused resources make applications larger and slow down builds.